core/src/main/java/nl/nn/adapterframework/jdbc/dbms/MariaDbDbmsSupport.java

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright 2019 Nationale-Nederlanden, 2020 WeAreFrank!
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package nl.nn.adapterframework.jdbc.dbms;
+
+import java.sql.Connection;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import org.apache.commons.lang.StringUtils;
+
+import nl.nn.adapterframework.jdbc.JdbcException;
+import nl.nn.adapterframework.util.JdbcUtil;
+
+/**
+* Support for MySql/MariaDB.
+* 
+*/
+public class MariaDbDbmsSupport extends GenericDbmsSupport {
```

Comment:
Would it be possible to let MariaDbDbmsSupport to extend MySqlDbmsSupport instead of GenericDbmsSupport?